### ISSUE TITLE
PHP fatal error accessing DataEntry form when "Allow events to be filled out independently from each other:" is off

### ISSUE THREAD

**Issue by pbchase** [ctsit/linear_data_entry_workflow#40]
As I loaded a DataEntry form, I got this message: 

```
REDCap crashed due to an unexpected PHP fatal error!

Error message: Uncaught Error: Unsupported operand types in
/vagrant/modules/linear_data_entry_workflow_v0.0.0/ExternalModule.php:110
Stack trace: #0
/vagrant/modules/linear_data_entry_workflow_v0.0.0/ExternalModule.php(58): Lin
earDataEntryWorkflow\ExternalModule\ExternalModule->loadRFIO('data_entry_form'
, '1', '5', '80', 'icf') #1 [internal function]: LinearDataEntryWorkflow\Exter
nalModule\ExternalModule->hook_data_entry_form('25', '5', 'icf', '80', '9', 1)
#2 /var/www.backup/redcap-
20180807T1535/redcap_v8.4.4/ExternalModules/classes/ExternalModules.php(1294):
call_user_func_array(Array, Array) #3 /var/www.backup/redcap-
20180807T1535/redcap_v8.4.4/ExternalModules/classes/ExternalModules.php(1377):
ExternalModules\ExternalModules::startHook('linear_data_ent...', 'v0.0.0',
Array) #4 /var/www.backup/redcap-
20180807T1535/redcap_v8.4.4/Classes/Hooks.php(37):
ExternalModules\ExternalModules::callHook('data_entry_form', Array) #5
/var/www.backup/redcap-20180807T1535/redcap_v8.4.4/DataEntry/index.php(1708):
Hooks::call('redcap_dat

File: /vagrant/modules/linear_data_entry_workflow_v0.0.0/ExternalModule.php
Line: 110
```

This is the module configuration in use:

![screen shot 2018-08-09 at 4 32 52 pm](https://user-images.githubusercontent.com/3356493/43924190-f0529b22-9bf1-11e8-89b0-9a238294531d.png)

Reactivating "Allow events to be filled out independently from each other:" prevents the error.

**Comment by tbembersimeao** [comment]
For bug reproduction purposes: it happens only when the accessed form is not in the first event.

**Comment by pbchase** [comment]
Fixed in PR #41
